src/librustdoc/lib.rs

```diff
@@ -536,6 +536,7 @@ fn opts() -> Vec<RustcOptGroup> {
                 "[unversioned-shared-resources,toolchain-shared-resources,invocation-specific]",
             )
         }),
+        unstable("no-run", |o| o.optflag("", "no-run", "Compile, but don't run doc tests")),
```

Comment:
~~This option is in rustdoc, compile means render rust doc?~~

```suggestion
        unstable("no-run", |o| o.optflag("", "no-run", "Compile doc tests without running it")),
```

Will this be better?